src/pipelines/epidemiology/config.yaml

```diff
@@ -479,21 +484,6 @@ sources:
     test:
       metadata_query: "key.str.match('NO_.*')"
 
-
-  # Data sources for PE level 2
-  - name: pipelines.epidemiology.xx_wikipedia.WikipediaDataSource
-    fetch:
-      - url: "https://es.wikipedia.org/wiki/Pandemia_de_enfermedad_por_coronavirus_de_2020_en_Per%C3%BA"
```

Comment:
I looked into this a little bit, it's odd but the wikipedia page is still up and the table looks up-to-date. https://es.wikipedia.org/wiki/Pandemia_de_enfermedad_por_coronavirus_de_2020_en_Per%C3%BA

Reply:
It could be a number of things, including other tables which are making the parser break. The Wikipedia article parser is finicky (especially for locales other than en_US) so I'm happy to remove it since we have better sources available now.